src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx

```diff
@@ -111,7 +112,8 @@ const ProfileTabForm: FC<ProfileTabFormProps> = ({
 
   return (
     <Box sx={styles.profileGeneralTabContainer}>
-      <Box sx={styles.avatar.root}>
+      <Box id='photo' sx={styles.avatar.root}>
```

Comment:
For what reason you use this `id="photo"`?

Reply:
Hi, I use it to declare component that needs to be highlighted when user clicks on certain field in Complete Profile